docs/tasks.md

```diff
@@ -154,7 +163,7 @@ Updates CMR metadata to set access constraints
 
 ### [@cumulus/update-granules-cmr-metadata-file-links](https://github.com/nasa/cumulus/tree/master/tasks/update-granules-cmr-metadata-file-links)
 
-Update CMR metadata files with correct online access urls and etags and transfer etag info to granules' CMR files
+Update cmr metadata files with correct online access urls and etags and transfer etag info to granules' CMR files
```

Comment:
I know you didn't change this but I am curious why this is lowercase now

Reply:
This is auto-generated from the package.json for the task:

https://github.com/nasa/cumulus/blob/feature-rds/tasks/update-granules-cmr-metadata-file-links/package.json#L4

So this got changed when I re-generated the docs, as I did locally when making some docs. I can update the package.json and re-generate though

Reply:
https://github.com/nasa/cumulus/pull/2053/commits/7b1cfdab46b68976d2eff97c4015ac43d80000c9